apps/server/src/modules/board/repo/recursive-delete.visitor.spec.ts

```diff
@@ -441,4 +442,28 @@ describe(RecursiveDeleteVisitor.name, () => {
 			});
 		});
 	});
+
+	describe('visitCollaborativeTextEditorAsync', () => {
+		const setup = () => {
+			const childCollaborativeTextEditorElement = collaborativeTextEditorElementFactory.build();
+			const collaborativeTextEditorElement = collaborativeTextEditorElementFactory.build({
+				children: [childCollaborativeTextEditorElement],
```

Comment:
This contradicts that the element is not allowed to have children.

Reply:
Low priority.